src/pods/canvas/canvas.vm.ts

```diff
@@ -47,3 +47,14 @@ export interface XRelationCoords {
   xOrigin: number;
   xDestination: number;
 }
+
+export interface YRelationCoords {
+  yOrigin: number;
+  yDestination: number;
+}
+
+export interface seekResult {
```

Comment:
We can remove _seekResult_ from this file

one thing to take into account interfaces should be always uppercase

```diff
- export interface seekResult {
+ export interface SeekResult {
```

Reply:
It's fixed